src/redux/unleash/unleashSelectors.js

```diff
@@ -0,0 +1,13 @@
+
+export const initialState = {
+    unleash: null,
+    unleashLoading: true,
+    unleashError: undefined
+};
+
+export const getUnleash = (state = initialState) => state.unleash ? state.unleash.unleash : null;
+export const getUnleashLoading = (state = initialState) => state.unleash ? state.unleash.unleashLoading : true;
+export const getUnleashError = (state = initialState) => state.unleash ? state.unleash.unleashError : undefined;
+
```

Comment:
Kanskje lage en selector som tar inn navn på toggle som parameter og returnerer status?
getToggle(<toggle-name>) - så trenger vi ikke hente alle overalt i koden.

Reply:
Feil her - en unleash" for mye - skal være:
`export const getToggleStatus = (toggle, state = initialState) => state.unleash ? state.unleash.unleash[toggle] : false;`

Så ville jeg byttet om å inn-parametre: `(state = initialState, toggle)` - så blir det likt som andre steder